libs/ui-lib/lib/ocm/components/clusterConfiguration/staticIp/components/YamlView/yamlViewValidationSchema.tsx

```diff
@@ -11,26 +11,18 @@ import {
   UniqueStringArrayExtractor,
 } from '../../commonValidationSchemas';
 import {
+  FILE_TYPE_MESSAGE,
   getMaxFileSizeMessage,
-  isStringValidJSON,
-  isStringValidYAML,
   validateFileSize,
+  validateFileType,
 } from '../../../../../../common/utils';
 
-const MAX_FILE_SIZE = 100000; //100 kb
 const requiredMsg = 'A value is required';
-const FILE_TYPE_MESSAGE = 'Unsupported file type. Please provide a valid YAML file.';
 
 const networkYamlValidationSchema = Yup.string()
   .required(requiredMsg)
-  .test(
-    'file-type-yaml',
-    FILE_TYPE_MESSAGE,
-    (value: string) => isStringValidYAML(value) && !isStringValidJSON(value),
-  )
-  .test('file-size-limit', getMaxFileSizeMessage(MAX_FILE_SIZE), (value: string) =>
-    validateFileSize(value, MAX_FILE_SIZE),
-  );
+  .test('file-size-limit', getMaxFileSizeMessage(), validateFileSize)
+  .test('file-type-yaml', FILE_TYPE_MESSAGE, (value: string) => validateFileType(value));
```

Comment:
If we are using a function that allows yaml and json (which is correct in this case - it was wrong before), we should use an appropriate validation message. You should be able to reuse the `INCORRECT_TYPE_FILE` string.